futurex_openedx_extensions/helpers/theme.py

```diff
@@ -0,0 +1,31 @@
+""""Theme helpers"""
+from typing import Any, Optional
+
+from common.djangoapps.student.models import CourseAccessRole
+from django.conf import settings
+
+from futurex_openedx_extensions.helpers.roles import get_accessible_tenant_ids
+from futurex_openedx_extensions.helpers.tenants import get_all_tenants_info
+from futurex_openedx_extensions.helpers.users import is_system_staff_user
+
+
+def get_fx_dashboard_url(request: Any) -> Optional[str]:
+    """
+    Get Fx Dashboard URL
+
+    :param request: Current django request.
+    :type request: Request
+    :return: Fx Dashboard URL
+    :rtype: str
+    """
+    if (
+        is_system_staff_user(request.user) or
+        CourseAccessRole.objects.filter(user_id=request.user.id).exists()
+    ):
+        lang = 'en' if request.LANGUAGE_CODE != 'ar' else 'ar'
```

Comment:
@tehreem-sadat I asked for `en` as default, but Omar recommends `ar` as default (which makes sense because the staff are mostly Arabs)
```suggestion
        lang = 'en' if request.LANGUAGE_CODE == 'en' else 'ar'
```